### PR TITLE
fix: ssl.keystore.type=pkcs12

### LINKED ISSUE #6064: KSQLDB fails to start if ssl.keystore.type=pkcs12 instead of ssl.keystore.type=PKCS12
**Describe the bug**

KSQLDB fails to start if ssl.keystore.type=pkcs12 instead of ssl.keystore.type=PKCS12

**To Reproduce**
Steps to reproduce the behavior, include:
1. The version of KSQL.
2. Sample source data.
3. Any SQL statements you ran

Latest version of KSQLDB.

**Expected behavior**

KSQLDB should start even if  `ssl.keystore.type=pkcs12`, case sensitivity should be irrelevant for this property.

**Actual behaviour**
A clear and concise description of what actually happens, including:
1. CLI output
2. Error messages
3. KSQL logs

Essentially if you have ssl.keystore.type=pkcs12 it fails with the following stack:

[2020-08-20 11:42:09,539] ERROR Failed to start KSQL (io.confluent.ksql.rest.server.KsqlServerMain:66)
org.apache.kafka.common.config.ConfigException: Invalid value pkcs12 for configuration ssl.keystore.type: String must be one of: JKS, PKCS12
        at org.apache.kafka.common.config.ConfigDef$ValidString.ensureValid(ConfigDef.java:970)
        at org.apache.kafka.common.config.ConfigDef.parseValue(ConfigDef.java:512)
        at org.apache.kafka.common.config.ConfigDef.parse(ConfigDef.java:496)
        at org.apache.kafka.common.config.AbstractConfig.<init>(AbstractConfig.java:108)
        at org.apache.kafka.common.config.AbstractConfig.<init>(AbstractConfig.java:129)
        at io.confluent.ksql.rest.server.KsqlRestConfig.<init>(KsqlRestConfig.java:571)
        at io.confluent.ksql.rest.server.KsqlServerMain.createExecutable(KsqlServerMain.java:116)
        at io.confluent.ksql.rest.server.KsqlServerMain.main(KsqlServerMain.java:59)

**Additional context**

Other components in the Kafka ecosystem which use this same property do not have this case sensitivity issue.

### PR DESCRIPTION
**Pull request by uurl**
### Description 
This PR fixes #6064 KSQLDB fails to start if ssl.keystore.type=pkcs12 instead of ssl.keystore.type=PKCS12 

### Testing done 
mvn clean package 

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/Server.java [hunk 1]
```diff
@@ -331,7 +331,7 @@ private static void setTlsOptions(
       } else if (keyStoreType.equals(KsqlRestConfig.SSL_STORE_TYPE_JKS)) {
```
Comment (spena) [inline ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/Server.java:331]: Could you also use equalsIgnoreCase() here? We probably need it too.

**Review by spena — COMMENTED** [review]
Thanks @uurl for the quick patch.

Could you also change the config validator in the `KsqlRestConfig` to be case insensitive?
from:
```
public static final ConfigDef.ValidString SSL_STORE_TYPE_VALIDATOR =
      ConfigDef.ValidString.in(
          SSL_STORE_TYPE_JKS,
          SSL_STORE_TYPE_PKCS12
      );
```
to:
```
public static final ConfigDef.CaseInsensitiveValidString SSL_STORE_TYPE_VALIDATOR =
      ConfigDef.CaseInsensitiveValidString.in(
          SSL_STORE_TYPE_JKS,
          SSL_STORE_TYPE_PKCS12
      );
```

File: ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/Server.java (hunk 1 above)
Comment (uurl) [reply ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/Server.java]: Ready @spena

Comment (spena) [reply ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/Server.java]: Cool, thanks. Btw, the `KsqlConfig` has several changes, most of them are tab-spaces. Could you fix them too?

**Comment by spena** [comment]
@uurl I see too many changes in the KsqlConfig. Could be that you're using a different branch and attempting to merge into master?

**Pull request closed without merging**